README.md

```diff
@@ -1,22 +1,26 @@
 # OpenZeppelin Contracts for Cairo
+
 [![Tests and linter](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/python-app.yml/badge.svg)](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/python-app.yml)
 
 **A library for secure smart contract development** written in Cairo for [StarkNet](https://starkware.co/product/starknet/), a decentralized ZK Rollup.
 
 ## Usage
 
 > ## ⚠️ WARNING! ⚠️
+
 > This is repo contains highly experimental code.
```

Comment:
```suggestion
> ## ⚠️ WARNING! ⚠️
> This repo contains highly experimental code.
```

I think the display without the blank line looks more cohesive.

Reply:
The rendered Markdown looks the same with / without these newlines. But I reverted to avoid churn if you prefer the blank lines when reading the raw Markdown.